index.js

```diff
@@ -4,12 +4,12 @@ var utils = require('./utils')
 
 // order by descending value, minus the inputs approximate fee
 function utxoScore (x, feeRate) {
-  return x.value - (feeRate * utils.inputBytes(x))
+  return x.value.sub((feeRate.mul(utils.inputBytes(x))))
 }
 
 module.exports = function coinSelect (utxos, outputs, feeRate) {
-  utxos = utxos.concat().sort(function (a, b) {
-    return utxoScore(b, feeRate) - utxoScore(a, feeRate)
+  utxos = utxos.concat().sort((a, b) => {
```

Comment:
Please stick to `function()` for compatibility with old browsers (e.g. marshmallow). This library is not transpiled